examples/surface_timeseries.py

```diff
@@ -0,0 +1,33 @@
+"""
+Display a surface timeseries using data from nilearn
+"""
+
+try:
+    from nilearn import datasets
+    from nilearn import surface
+except ImportError:
+    raise ImportError("""This example uses data and methods from nilearn but
+    nilearn installed. To install try 'pip install nilearn'.""")
```

Comment:
```suggestion
    nilearn is not installed.""")
```

(a) the string formatting is a bit weird. I think with my suggestion you might be able to fit it in its own line with:

```python
raise ImportError(
    'This example uses ... but nilearn is not installed.'
)
```

(b) I don't want to prescribe installation methods here. Currently mixing pip and conda is discouraged, so, I prefer that users use their best judgment/google how to install nilearn.

Reply:
have improved formatting and wording. have dropped pip reference